docs/content/NavList.mdx

```diff
@@ -0,0 +1,536 @@
+        <li><a href="/actions">General</a></li>
+        <li><a href="/actions/runners">Runners</a></li>
+      </ul>
+    </li>
+  </ul>
+</nav>
+```
+
+</details>
+
+### With a divider
+
+```jsx
+<NavList>
+  <NavList.Item href="/">Dashboard</NavList.Item>
+  <NavList.Item href="/pulls">Pull requests</NavList.Item>
+  <NavList.Item href="/issues">Issues</NavList.Item>
+  <NavList.Divider />
+  <NavList.Item href="/marketplace">Marketplace</NavList.Item>
+  <NavList.Item href="/explore">Explore</NavList.Item>
+</NavList>
+```
+
+<details>
+  <summary>Rendered HTML</summary>
+
+```html
+<nav>
+  <ul role="list">
+    <li><a href="/">Dashboard</a></li>
+    <li><a href="/pulls">Pull requests</a></li>
+    <li><a href="/issues">Issues</a></li>
+    <div role="separator"></div>
+    <li><a href="/marketplace">Marketplace</a></li>
+    <li><a href="/explore">Explore</a></li>
+  </ul>
+</nav>
+```
+
+</details>
+
+### With React Router
+
+```jsx
+import {Link, useMatch, useResolvedPath} from 'react-router-dom'
+import {NavList} from '@primer/react'
+
+function NavItem({to, children}) {
+  const resolved = useResolvedPath(to)
+  const isCurrent = useMatch({path: resolved.pathname, end: true})
+  return (
+    <NavList.Item as={Link} to={to} current={isCurrent}>
+      {children}
+    </NavList.Item>
+  )
+}
+
+function App() {
+  return (
+    <NavList>
+      <NavItem to="/">Dashboard</NavItem>
+      <NavItem to="/pulls">Pull requests</NavItem>
+      <NavItem to="/issues">Issues</NavItem>
+    </NavList>
+  )
+}
+```
+
+### With Next.js
+
+```jsx
+import {useRouter} from 'next/router'
+import Link from 'next/link'
+import {NavList} from '@primer/react'
+
+function NavItem({href, children}) {
+  const router = useRouter()
+  const isCurrent = router.asPath === href
+  return (
+    <Link href={href} passHref>
+      <NavList.Item current={isCurrent}>{children}</NavList.Item>
+    </Link>
+  )
+}
+
+function App() {
+  return (
+    <NavList>
+      <NavItem href="/">Dashboard</NavItem>
+      <NavItem href="/pulls">Pull requests</NavItem>
+      <NavItem href="/issues">Issues</NavItem>
+    </NavList>
+  )
+}
+```
+
+## Props
+
+### NavList
+
+<PropsTable>
+  <PropsTableRow name="aria-label" type="string" />
+  <PropsTableRow name="aria-labelledby" type="string" />
+  <PropsTableSxRow />
+  <PropsTableRefRow refType="HTMLElement" />
+</PropsTable>
+
+### NavList.Item
+
+<PropsTable>
+  <PropsTableRow
+    name="href"
+    type="string"
+    description={
+      <>
+        The URL that the item navigates to. <InlineCode>href</InlineCode> is passed to the underlying{' '}
+        <InlineCode>&lt;a&gt;</InlineCode> element. If <InlineCode>as</InlineCode> is specified, the component may need
+        different props. If the item contains sub-items, the item is rendered as a{' '}
+        <InlineCode>&lt;button&gt;</InlineCode> and <InlineCode>href</InlineCode> is ignored.
+      </>
+    }
+  />
+  <PropsTableRow
+    name="current"
+    type="boolean"
```

Comment:
`aria-current` can be `page` or `location` (string)

Reply:
When would we use `page` vs `location`?

Reply:
From [MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current#values):

> `location`:
> Represents the current location within an environment or context such as the image that is visually highlighted as the current component of a flow chart.

I'm curious when we'd use NavList to link to a "location," as defined above.

Reply:
`location` would be used for hash links. Here's a nice [definition](https://www.digitala11y.com/aria-current-state/):

> page = Represents the current page within a set of pages
> location = Represents the current location within an environment or context

There are more options for `aria-current`, but I think these are the only two that make sense for a navigational component.

Reply:
Ah I see. Thanks for explaining! So, for example, the current link in a table of contents would have `aria-current="location"`?

I wonder if it makes sense to just have this component accept `aria-current` directly, instead of abstracting it behind a `current` prop. What do you think?

Reply:
> the current link in a table of contents would have aria-current="location"

Yup! In some of the Settings pages we link to subheadings, and we still want to show that "page" as active even if the link in the NavList is linking to #some-id

> I wonder if it makes sense to just have this component accept aria-current directly, instead of abstracting it behind a current prop

I think that makes more sense too. I wonder if there's a way to help the end user decide which option to use (page or location) since I think its a lesser known aria label. Or maybe we can infer it based on what kind of link is passed in?

Reply:
I removed the `current` prop in favor of the native `aria-current` prop 👍